scripts/modules/commands.js

```diff
@@ -132,13 +145,709 @@ import {
+ */
+function isConnectedToHamster() {
+	try {
+		const authManager = AuthManager.getInstance();
+		const context = authManager.getContext();
+
+		// Check if user has a brief context
+		if (context && context.briefId) {
+			return true;
+		}
+
+		// Fallback: Check if storage type is 'api' (user selected Hamster during init)
+		try {
+			const config = getConfig();
+			if (config?.storage?.type === 'api') {
+				return true;
+			}
+		} catch {
+			// Config check failed, continue
+		}
+
+		return false;
+	} catch {
+		return false;
+	}
+}
+
+/**
+ * Prompt user about using Hamster for collaborative PRD management
+ * Only shown to users who are not already connected to Hamster
+ * @returns {Promise<'local'|'hamster'>} User's choice
+ */
+async function promptHamsterCollaboration() {
+	// Skip prompt in non-interactive mode only
+	if (!process.stdin.isTTY) {
+		return 'local';
+	}
+
+	console.log(
+		'\n' +
+			chalk.bold.white(
+				'Your tasks are only as good as the context behind them.'
+			) +
+			'\n\n' +
+			chalk.dim(
+				'Parse locally and tasks will be stored in a JSON file. Bring it to Hamster and your brief\nbecomes part of a living system connected to your team, your codebase and your agents.\nNow your entire team can go as fast as you can with Taskmaster.'
+			) +
+			'\n'
+	);
+
+	const { choice } = await inquirer.prompt([
+		{
+			type: 'list',
+			name: 'choice',
+			message: chalk.cyan('How would you like to parse your PRD?'),
+			choices: [
+				'\n',
+				{
+					name: [
+						chalk.bold('Parse locally'),
+						'',
+						chalk.white(
+							'   • Your PRD becomes a task list in a local JSON file'
+						),
+						chalk.white(
+							'   • Great for quick prototyping and for vibing on your own'
+						),
+						chalk.white('   • You can always export to Hamster later'),
+						''
+					].join('\n'),
+					value: 'local',
+					short: 'Parse locally'
+				},
+				{
+					name: [
+						chalk.bold('Bring it to Hamster'),
+						'',
+						chalk.white(
+							'   • Your PRD will become a living brief you can refine with your team'
+						),
+						chalk.white(
+							'   • Hamster will generate tasks automatically, ready to execute in Taskmaster'
+						),
+						chalk.white(
+							'   • Hamster will automatically analyze complexity and expand tasks as needed'
+						),
+						chalk.white(
+							'   • Invite your teammates to collaborate on a single source of truth'
+						),
+						chalk.white(
+							'   • AI inference handled by Hamster, no API keys needed - just a Hamster account!'
+						),
+						''
+					].join('\n'),
+					value: 'hamster',
+					short: 'Bring it to Hamster'
+				}
+			],
+			default: 'local',
+			pageSize: 20
+		}
+	]);
+
+	return choice;
+}
+
+/**
+ * Handle parsing PRD to Hamster
+ * Creates a brief from the PRD content and sets context
+ * @param {string} prdPath - Path to the PRD file
+ */
+async function handleParsePrdToHamster(prdPath) {
```

Comment:
please move to app/cli, also I didn't look at the specifics, but it seems like this is something you're already doing a version of in parse-prd-to-hamster.ts ?